xbuild/src/cargo/mod.rs

```diff
@@ -465,7 +465,7 @@ impl CargoBuild {
     }
 
     pub fn set_sysroot(&mut self, path: &Path) {
-        let arg = format!("--sysroot=\"{}\"", path.display());
```

Comment:
I'm not entirely sure why adding quotations here breaks it for me on Windows. I printed out the provided path which returned the following:`

```
C:\Users\mdans\AppData\Local\x\Android.ndk
```

This seems correct to me 👀